pkg/motor/login.go

```diff
@@ -56,8 +57,7 @@ func (mtr *motorNodeImpl) Login(request mt.LoginRequest) (mt.LoginResponse, erro
 	mtr.sharedShard = shards.PskShard
 	mtr.recoveryShard = shards.RecoveryShard
 	mtr.unusedShards = destructureShards(shards.ShardBank)
-	mtr.callback.OnMotorEvent(ct.MotorCallbackMessage_MTR_LOGGED_IN, true)
-
+	mtr.callback.OnMotorEvent("Logged into account successfully!", true)
```

Comment:
References to the value represented by each string should be made through some kind of strongly typed value / constant